src/Atmos/Model/linear.jl

```diff
@@ -0,0 +1,63 @@
+abstract type AtmosLinearModel <: BalanceLaw
+end
+
+
+vars_state(lm::AtmosLinearModel, T) = vars_state(lm.atmos,T)
+vars_gradient(lm::AtmosLinearModel, T) = @vars()
+vars_diffusive(lm::AtmosLinearModel, T) = @vars()
+vars_aux(lm::AtmosLinearModel, T) = vars_aux(lm.atmos,T)
+vars_integrals(lm::AtmosLinearModel,T) = @vars()
+
+
+update_aux!(dg::DGModel, lm::AtmosLinearModel, Q::MPIStateArray, auxstate::MPIStateArray, t::Real) = nothing
+integrate_aux!(lm::AtmosLinearModel, integ::Vars, state::Vars, aux::Vars) = nothing
+flux_diffusive!(lm::AtmosLinearModel, flux::Grad, state::Vars, diffusive::Vars, aux::Vars, t::Real) = nothing
+function wavespeed(lm::AtmosLinearModel, nM, state::Vars, aux::Vars, t::Real)
+  ref = aux.ref_state
+  return soundspeed_air(ref_state.T)
+end
+
+boundary_state!(nf, lm::AtmosLinearModel, x...) = nothing
+init_aux!(lm::AtmosLinearModel, aux::Vars, geom::LocalGeometry) = nothing
+init_state!(lm::AtmosLinearModel, state::Vars, aux::Vars, coords, t) = nothing
+
+
+struct AtmosAcousticLinearModel{M} <: AtmosLinearModel
+  atmos::M
+end
+function flux_nondiffusive!(lm::AtmosAcousticLinearModel, flux::Grad, state::Vars, aux::Vars, t::Real)
+  DF = eltype(state)
+  ref = aux.ref_state
+  e_pot = gravitational_potential(lm.atmos.orientation, aux)
+
+  flux.ρ = state.ρu
+  # TODO: use MoistThermodynamics.linearized_air_pressure 
+  # need to avoid dividing then multiplying by ρ
+  pL = state.ρ * DF(R_d) * DF(T_0) + DF(R_d) / DF(cv_d) * (state.ρe - state.ρ * e_pot)
```

Comment:
@charleskawczynski provided a function `linearized_air_pressure` in `MoistThermodynamics` for this. It seems preferable to me to use that.

Reply:
Yes, see the comment. There were some issues in that it divides then multiplying by ρ.

Reply:
I'll add a function in MoistThermo that doesn't require the multiply-divide